src/Controllers/FilesController.js

```diff
@@ -103,9 +118,9 @@ export class FilesController {
     }
   }
 
-  getExpressRouter() {
+  static getExpressRouter() {
```

Comment:
@nlutsenko moved here to a static router to enable controller switching at runtime instead of a route/app.